### ISSUE TITLE
RA stopped after preconstruction stage

### ISSUE THREAD

**Issue by vmurigneu** [rvaser/ra#11]
Hi Robert,

I have installed RA following instructions without errors. 
But when I tried to run the following command:
python /sw/ra/build/bin/ra -t 16 -x ont ont.fastq.gz > ra.fa

I get an empty assembly fasta file and the following message: 
[Ra::run] preconstruction overlap stage
[M::mm_idx_gen::185.227*2.22] collected minimizers
[Ra::__exit__] warning: unable to clean work directory!

Any idea? 

Thanks in advance. 
Valentine

**Comment by rvaser** [comment]
Hello Valentine,
looking at the log I suspect that minimap crashed due to insufficient memory. How much RAM do you have? How large are your reads?

Best regards,
Robert

**Comment by vmurigneu** [comment]
Yes it is very likely to have insufficient memory. I have 31G and  around 43G of compressed of human data. I am trying to run Ra on a larger memory node (120G).

Thanks.

**Comment by rvaser** [comment]
Hmmm, not sure what minimap requirement is, but it seems to me that 120GB should be enough.
